upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -109,6 +109,13 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 		return fmt.Sprintf("%d", wellknownports.NodeLocalDNSHealthCheck)
 	}
 
+	dest["KopsControllerImage"] = func() string {
```

Comment:
similar to the etcd-manager image override, I'm wondering if we need this permanently or not. its possibly some release scripts might need updating if they automatically bump the manifest's kops-controller image tag. @hakman might know

Reply:
This was probably needed for testing. I see no need for it or the etcd manager override. In fact I also wanted to ask to remove them both from the PR.